src/test/deployments.test.js

```diff
@@ -197,7 +197,6 @@ describe("Deployments navigation badge", function () {
 
   it("has the correct amount of deployments", function () {
     var badge = this.component.props.children[1].props.children[1];
-    expect(badge.props.className).to.equal("badge");
```

Comment:
@aldipower this guy was responsible for the failing tests. Do you agree that this line really adds no value to the test and thus should be removed?